src/cryptsetup/cryptsetup.c

```diff
@@ -1129,7 +1129,6 @@ static int attach_luks_or_plain_or_bitlk_by_tpm2(
                         if (r != -EAGAIN) /* EAGAIN means: no tpm2 chip found */
                                 return r;
                 }
-                assert(decrypted_key);
```

Comment:
i cherry picked this commit into the main branch, since it's obvious. please rebase.